src/singletons/Settings.hpp

```diff
@@ -643,6 +643,9 @@ class Settings
         false,
     };
 
+    QStringSetting additionalExtensionIDs{"/appearance/additionalExtensionIDs",
```

Comment:
Change the JSON path from `/appearance` to `/misc` - this has no functional difference but moves where it'll be stored in the  `settings.json` file
```suggestion
    QStringSetting additionalExtensionIDs{"/misc/additionalExtensionIDs",
```